src/api/views.py

```diff
@@ -32,7 +32,9 @@
         "Read more about choosing your .gov domain.</a>".format(public_site_url("domains/choosing"))
     ),
     "invalid": "Enter a domain using only letters, numbers, or hyphens (though we don't recommend using hyphens).",
-    "success": "That domain is available!",
+    "success": "That domain is available! We’ll try to give you the domain you want, "
```

Comment:
Had some time to look into the linting error: First thing that came to mind is that DOMAIN_API_MESSAGES is a 1-to-1 key-value dictionary that maps a message-category to the message content text. The code interpreter/compiler is likely confused because it's reading the mapping for success message-category as 3 items (for the 3 lines we added to accommodate our new longer success text) when each message category is only supposed to have 1 item assigned to it. Most likely this is what's causing the linter to fail (and would also lead to this message erroring when we try passing it into the UI possibly). 

**Solution**
When we want to write a single text string that consists of multiple lines of code, I put a backwards slash at the end of the line to indicate to the code interpreter that my I want my single string to include what's on the next line ([More info here](https://note.nkmk.me/en/python-long-string/#:~:text=In%20Python%2C%20a%20backslash%20(%20%5C,continue%20on%20the%20next%20line.)). I can push an update to this branch to update this!

Reply:
Pushing a change that updates the following API message:
**Before**
```
"success": "That domain is available! We’ll try to give you the domain you want, "
           "but it's not guaranteed. After you complete this form, we’ll "
           "evaluate whether your request meets our requirements.",
```

**After**
```
"success": "That domain is available! We’ll try to give you the domain you want, \
               but it's not guaranteed. After you complete this form, we’ll \
               evaluate whether your request meets our requirements.",
```

Reply:
@erinysong Thank you!!!!!